README.md

```diff
@@ -25,36 +25,38 @@ Draft information for our presentation.
     - Are the factors that influence the length of stay different for different animals?
 
   
-# Austin AniML Rescue communication protocol
 
-## To ensure that the group receive consistent messages, we create a communication protocol outlines, where we identify the person responsibilities inside the final Bootcamp project.
+# Austin AniML Rescue CommunicationProtocol
 
+## To ensure that the group receive consistent messages, we create a communication protocol outlines, where we identify the person responsibilities inside the final Bootcamp project.
 ### Everyone in the Austin AniML Rescue project has a role to play in communication:
-             •	Ilaha Mammadzada (square) is the manager and responsible for the repository.
-             •	Melinda Malone (circle) is in charge of the merging and cleaning of the data set, 
+
+     - Ilaha Mammadzada(square) is the manager and responsible for the repository.
+     - Melinda Malone (circle) is in charge of the merging and cleaning of the data set, 
                 to make sure the database will work seamlessly with the rest of the project
-             •	Sarah Griffin (X) are in charge of deciding which technologies will be used for 
+     - Sarah Griffin (X) are in charge of deciding which technologies will be used for 
                 each step of the project as well as working in machine learning model.
-             •	Arlena Wu  (triangle) is in charge to create a sample of a machine leaning 
+     - Arlena Wu  (triangle) is in charge to create a sample of a machine leaning 
                 model for the project. Where she show and explain how it will work with the rest of the project steps.
-             •	Ruth Ortiz (x) and (triangle) is in charge to create the diagram that explains how it the project works
+     - Ruth Ortiz(x) and (triangle) is in charge to create the diagram that explains how it the project works
                 by creating a visualization illustration in tableau.
             
-## Everybody is responsible to voice their concerns and issues, provide feedbacks and listen to the other effectively.
```

Comment:
Line 43 - Consider changing to:  Providing feedback on Group Work
Line 44 - 46:
- After committing work to the appropriate branch, submit a pull request once you believe the work is ready to be merged to the main branch.
- Two additional team members need to review the work as soon as possible and leave comments.
- A third team member completes the final review and merges the changes. This should be the person serving as the Repository manager for that week of the project.
- If significant changes are required to the commit, it is advised to ask the original author to revise the work and submit a new commit. Then the process starts again.